notification_groups_test.go

```diff
@@ -0,0 +1,292 @@
+package mackerel
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/kylelemons/godebug/pretty"
```

Comment:
[memo] `pretty` is already used in **downtimes_test.go**, and **monitors_test.go**.